src/utils/numbers/units.json

```diff
@@ -0,0 +1,30 @@
+{
+  "wei":          "1",
+  "kwei":         "1000",
+  "Kwei":         "1000",
```

Comment:
Why we can't just write keys in lowercase and just capitalize they in code?
Also, why we save this map in json?

Reply:
changed to two functions (fromWeiToGwei, fromGweiToWei), file removed